From de4fbfe93560c78f29c8b92cafab0793f5d26bc6 Mon Sep 17 00:00:00 2001
From: Aaron Ballman <aaron@aaronballman.com>
Date: Tue, 19 Jul 2016 20:20:03 +0000
Subject: [PATCH] This code block breaks the docs build
 (http://lab.llvm.org:8011/builders/llvm-sphinx-docs/builds/11921/steps/docs-llvm-html/logs/stdio).
 Setting the code highlighting to none instead of llvm to hopefully get the
 bot stumbling back towards green.

git-svn-id: https://llvm.org/svn/llvm-project/llvm/trunk@276018 91177308-0d34-0410-b5e6-96231b3b80d8
---
 docs/BitCodeFormat.rst | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/docs/BitCodeFormat.rst b/docs/BitCodeFormat.rst
index ffa2176..89c7c1b 100644
--- a/docs/BitCodeFormat.rst
+++ b/docs/BitCodeFormat.rst
@@ -596,7 +596,7 @@ will be encoded as 1.
 
 For example, instead of
 
-.. code-block:: llvm
+.. code-block:: none
 
   #n = load #n-1
   #n+1 = icmp eq #n, #const0
@@ -604,7 +604,7 @@ For example, instead of
 
 version 1 will encode the instructions as
 
-.. code-block:: llvm
+.. code-block:: none
 
   #n = load #1
   #n+1 = icmp eq #1, (#n+1)-#const0
-- 
2.5.5

